addons/calendar/models/calendar.py

```diff
@@ -1534,8 +1535,34 @@ def read(self, fields=None, load='_classic_read'):
             for k in EXTRAFIELDS:
                 if (k in r) and (fields and (k not in fields)):
                     del r[k]
+
+        tz = self.env.context and self.env.context.get('tz')
+        if tz:
+            tz = pytz.timezone(tz)
+        TIME_FIELDS_IN_EVENT = ['start', 'stop', 'start_datetime', 'stop_datetime']
+        for record in result:
+            if not record.get('allday') and isinstance(record['id'], (basestring)) and tz:
```

Comment:
```suggestion
            if not record.get('allday') and isinstance(record['id'], (basestring)) and tz and is_calendar_id(record['id']):
```
Otherwise you'll get an list index out of range error when deleting a normal meeting

Reply:
@migmarh I don't know where you get the `is_calendar_id` method, it is not defined on self and neither on my solution, also already tested deleting normal meetings and no problem there.

Reply:
It’s defined on line 61 of calendar.py in 10.0
As the read method gets called on unlink, it also passes for non recurring meetings

Reply:
I see, you are right, sorry i got caught up searching it in self, didn't notice it was a global method, code works great